docs/tutorials/kafka-streams-examples.rst

```diff
@@ -0,0 +1,247 @@
+        "plays": 70
+      },
+      {
+        "artist": "Hilltop Hoods",
+        "album": "The Calling",
+        "name": "The Calling",
+        "plays": 67
+      },
+
+      ... rest omitted...
+    ]
+
+The REST API exposed by the
+:cp-examples:`Kafka Music application|src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java`
+supports further operations.  See the
+:cp-examples:`top-level instructions in its source code|src/main/java/io/confluent/examples/streams/interactivequeries/kafkamusic/KafkaMusicExample.java`
+for details.
+
+Once you're done playing around you can stop all the services and containers with:
+
+.. sourcecode:: bash
+
+    $ docker-compose down
+
+We hope you enjoyed this tutorial!
+
+
+Running further Confluent demo applications for the Kafka Streams API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The container named ``kafka-music-application``, which runs the Kafka Music demo application, actually contains all of
+Confluent's `Kafka Streams demo applications <https://github.com/confluentinc/examples>`__.  The demo applications are
+packaged in the fat jar at ``/app/streams-examples-3.2.0-standalone.jar`` inside this container.  This means you can
+easily run any of these applications from inside the container via a command similar to:
+
+.. sourcecode:: bash
+
+    # Example: Launch the WordCount demo application (inside the `kafka-music-application` container)
+    $ docker-compose exec kafka-music-application \
+            java -cp /app/streams-examples-3.2.0-standalone.jar \
+            io.confluent.examples.streams.WordCountLambdaExample \
+            localhost:29092
+
+Of course you can also modify the tutorial's ``docker-compose.yml`` for repeatable deployments.
+
+Note that you must follow the full instructions of each demo application (see its respective source code at
+https://github.com/confluentinc/examples).  These instructions include, for example, the creation of the application's
+input and output topics.  Also, each demo application supports CLI arguments.  Typically, the first CLI argument is
+the ``bootstrap.servers`` parameter and the second argument, if any, is the ``schema.registry.url`` setting.
+
+Available endpoints **from within the containers**:
+
++---------------------------+-------------------------+---------------------------+
+| Endpoint                  | Parameter               | Value                     |
++===========================+=========================+===========================+
+| Kafka Cluster             | ``bootstrap.servers``   | ``localhost:29092``       |
++---------------------------+-------------------------+---------------------------+
+| Confluent Schema Registry | ``schema.registry.url`` | ``http://localhost:8081`` |
++---------------------------+-------------------------+---------------------------+
+| ZooKeeper ensemble        | ``zookeeper.connect``   | ``localhost:32181``       |
```

Comment:
The ZK endpoint is required to e.g. create topics manually, or when inspecting topic data with the console consumer.